backend/tracim_backend/error_code.py

```diff
@@ -0,0 +1,56 @@
+# Error code format : [TYPE:2c]-[NUMBER:2c]
+# TYPE:
+# NF=Not found error
+# RI=Right error
+# VA=validation post-marshallow
+# CO=Conflict error
+# AT=Auth error
+# MV = marshmallow schema validation error
+
+ERROR_CODE_GENERIC_SCHEMA_VALIDATION_ERROR = 'SV-01'
+# Tracim Not found Error
+ERROR_CODE_USER_NOT_FOUND = 'NF-01'
+ERROR_CODE_WORKSPACE_NOT_FOUND = 'NF-02'
+ERROR_CODE_CONTENT_NOT_FOUND = 'NF-03'
+# Preview Based
+ERROR_CODE_UNAIVALABLE_PREVIEW_TYPE = 'NF-11'
+ERROR_CODE_PAGE_OF_PREVIEW_NOT_FOUND = 'NF-12'
+ERROR_CODE_UNAIVALABLE_PREVIEW = 'NF-13'
+
+# Right Error
+ERROR_CODE_INSUFFICIENT_USER_PROFILE = 'RI-01'
+ERROR_CODE_INSUFFICIENT_USER_ROLE_IN_WORKSPACE = 'RI-02'
+
+# Auth Error
+ERROR_CODE_NOT_AUTHENTICATED = 'AT-01'
```

Comment:
We already have 401 for this case. Do nlt add a specific error code

Reply:
In fact, that's a bit more complicated. We have actually 3 error case who return 401 error.
ERROR_CODE_NOT_AUTHENTICATED is the most common one, it's for invalid user authentification.

With reset_password feature, we have now ERROR_CODE_INVALID_RESET_PASSWORD_TOKEN 
and
ERROR_CODE_EXPIRED_RESET_PASSWORD_TOKEN
who also return 401. 
That's maybe not the correct behaviour, but that's what actually done.

Reply:
The 2 last error cases are 400, not 401